crates/standalone/src/main.rs

```diff
@@ -46,6 +46,15 @@ use tikv_jemallocator::Jemalloc;
 #[global_allocator]
 static GLOBAL: Jemalloc = Jemalloc;
 
+// Defaults our jemalloc configuration to allow for profiling, but have it disabled initially.
+// It can be enabled with an internal endpoint.
+// When enabled, it will sample once per 2^19 bytes (512 KB) of memory allocated.
+
+// This can be overridden by setting the `_RJEM_MALLOC_CONF` environment variable.
+#[allow(non_upper_case_globals)]
+#[export_name = "_rjem_malloc_conf"]
```

Comment:
it looks like the `export_name` is the same as the actual name? what am I overlooking?

Reply:
I'm just following the pattern of their docs. The `export_name` part has something to do with how the jemalloc library looks it up.

Reply:
Can you add a comment linking to those docs?